tests/fixtures/analog/commands.js

```diff
@@ -0,0 +1,420 @@
+                dump: '0b 00',
+                parameters: {
+                    eventList: []
+                }
+            }
+        }
+    }
+};
+
+export const downlinkCommands = {
+    getCurrent: {
+        hex: {
+            dump: '07 00 52',
+            lrc: '0x52'
+        },
+        commands: {
+            getCurrent: {
+                name: 'getCurrent',
+                dump: '07 00'
+            }
+        }
+    },
+    getStatus: {
+        hex: {
+            dump: '14 00 41',
+            lrc: '0x41'
+        },
+        commands: {
+            getStatus: {
+                name: 'getStatus',
+                dump: '14 00'
+            }
+        }
+    },
+    getTime2000: {
+        hex: {
+            dump: '09 00 5c',
+            lrc: '0x5c'
+        },
+        commands: {
+            getTime2000: {
+                name: 'getTime2000',
+                dump: '09 00'
+            }
+        }
+    },
+    getBatteryStatus: {
+        hex: {
+            dump: '1f 05 00 4f',
+            lrc: '0x4f'
+        },
+        commands: {
+            getBatteryStatus: {
+                name: 'getBatteryStatus',
+                dump: '1f 05 00'
+            }
+        }
+    },
+    getLmicInfo: {
+        hex: {
+            dump: '1f 02 00 48',
+            lrc: '0x48'
+        },
+        commands: {
+            getLmicInfo: {
+                name: 'getLmicInfo',
+                dump: '1f 02 00'
+            }
+        }
+    },
+    'setParameter + getParameter + setParameter': {
+        hex: {
+            dump: '03 02 18 01 04 01 18 03 05 01 00 00 00 06 51',
+            lrc: '0x5e'
+        },
+        commands: {
+            setParameterAbsoluteData: {
+                name: 'setParameter',
+                dump: '03 02 18 01',
+                parameters: {
+                    id: 24,
+                    data: {
+                        state: 1
+                    }
+                }
+            },
+            getParameter: {
+                name: 'getParameter',
+                dump: '04 01 17',
+                parameters: {
+                    id: 24,
+                    data: null
+                }
+            },
+            setParameterInterval: {
+                name: 'setParameter',
+                dump: '03 05 01 00 00 00 06',
+                parameters: {
+                    id: 1,
+                    data: {
+                        state: 3600
+                    }
+                }
+            }
+        }
+    },
+    'getExAbsArchiveHoursMc + getExAbsArchiveDaysMc + getExAbsArchiveCurrentMc': {
+        hex: {
+            dump: '1f 0c 04 2f 97 0c 01 1f 0d 04 2f 98 01 01 1f 0f 00 46',
+            lrc: '0x46'
+        },
+        commands: {
+            getExAbsArchiveHoursMc: {
+                name: 'getExAbsArchiveHoursMc',
+                dump: '1f 0c 04 2f 97 0c 01',
+                parameters: {
+                    channelList: [
```

Comment:
in such simple cases it better be in one line

Reply:
fixed